qp/pdf.py

```diff
@@ -81,6 +122,10 @@ def plot(self, limits):
             y = [0., 1.]
             plt.vlines(self.quantiles, y[0], y[1], color='k', linestyle='--', lw=1.0, alpha=1., label='Quantiles')
 
+        if number is not None:
+            (grid, interpolated) = self.interpolate(number)
+            plt.plot(grid, interpolated, color='r', linestyle=':', lw=2.0, alpha=1.0, label='Interpolated PDF')
```

Comment:
Nice! Can you please paste the output of this plotting command to the PR thread? Thanks!